packages/host/src/QueryErrorHandler.tsx

```diff
@@ -16,7 +16,11 @@ export const QueryErrorHandler = () => {
     if (!!errorMessage) {
       error(errorMessage)();
     }
-  }, [errorMessage, location.pathname]);
+  }, [errorMessage]);
+
+  useEffect(() => {
```

Comment:
offroad.. when there was an error it was getting stuck and re-toasting when you navigated